Documentation/deploy-master.md

```diff
@@ -1,6 +1,6 @@
 # Deploy Kubernetes Master Node(s)
 
-Boot a single CoreOS machine which will be used as the Kubernetes master node. You must use a CoreOS version 773.1.0+ on the Alpha or Beta channel for the `kubelet` to be present in the image.
+Boot a single CoreOS machine which will be used as the Kubernetes master node. You must use a CoreOS version 960.0.0+ for the `kubelet-wrapper` script to be present in the image. If you wish to use an earlier version (e.g. from the 'stable' channel) see [kubelet-wrapper](kubelet-wrapper.md) for more information.
```

Comment:
I don't want to say that the kubelet is in the image (because we're going to be removing it). What about:

> "You must use a CoreOS version 962.0.0+ for the kubelet-wrapper script to be present in the image. See [kubelet-wrapper](kubelet-wrapper.md) for more information.